README.md

```diff
@@ -1,6 +1,6 @@
 # The polyPod
 
-At the moment, the polyPod is essentially a runtime for _features_; small,
+The polyPod aims to become a personal data store and secure execution environment for privacy-conscious computations. THe current iteration still lacks some of the desired functionality and is essentially a runtime for _Features_; small,
```

Comment:
You sure you want to say what it *lacks*?

Reply:
Seems fine to me. There's a typo though: `THe`. Capitalising _Feature_ seems like a good idea, think that makes it less confusing, but we'll need to update all the other READMEs where it's written in lower case, too.

Reply:
With "aims to become" it's already clear it's not there *yet*.